packages/react-scripts/scripts/new-component/templates/index.tmpl

```diff
@@ -36,7 +36,9 @@ export class {{ component }} extends Component {
 
   render() {
     return (
-      <div className="{{ component }}" />
+      <div className="{{ component }}" >
+        {{ component }}
+      </div>
     );
   }
 }
```

Comment:
Not related to a specific change in this PR, but I did notice that the `// eslint-disable-line-no-unused-vars` line below doesn't seem to be needed. If I take it out of the stock template, I don't get any linting errors.